src/cljc/matcher_combinators/standalone.cljc

```diff
@@ -3,14 +3,37 @@
             [matcher-combinators.core :as core]
             [matcher-combinators.parser]))
 
+(defn match
+  "Returns a map indicating whether the `actual` value matches the `matcher`.
+
+  `matcher` can be a matcher-combinators matcher, a predicate function of
+  actual, an expression that returns a value, or a literal value.
+
+  `actual` can be an expression that returns the actual value, or a literal.
+
+  Return map includes the following keys:
+
+  - :matcher-combinators.result/type
+    - either :match or :mismatch
+  - :matcher-combinators.result/value
+    - when :matcher-combinators.result/type is :match, binds to the actual value,
+      else a map indicating the expected and actual values "
+  [matcher actual]
+  (core/match matcher actual))
```

Comment:
```suggestion
  (dissoc (core/match matcher actual)) :matcher-combinators.result/weight)
```

Reply:
I would actually prefer an implementation that doesn't expose random internal representations from `core`

So something like the following feels a little more evolvable

```clojure
(let [match-results (core/match matcher actual)]
  [(= :match match-results)
   (:matcher-combinators.result/value match-results)])
```

maybe even making it a map with proper keys, if you can think of good names

Reply:
When the result is `:match`, the `:value` isn't that interesting. What do you think about this:

```clojure
;; match case
{:match/result :match}

;; mismatch case
{:match/result  :mismatch
 :match/details (:matcher-combinators.result/value match-results)}
```

Reply:
that sounds perfect!